example.js

```diff
@@ -0,0 +1,353 @@
+var mutate = require('xtend/mutable')
+var expose = require('choo-expose')
+var css = require('sheetify')
+var html = require('bel')
+var choo = require('./')
+
+css('todomvc-common/base.css')
+css('todomvc-app-css/index.css')
+
+if (module.parent) {
+  exports.todoStore = todoStore
+} else {
+  var app = choo()
+
+  if (process.env.NODE_ENV !== 'production') {
+    var persist = require('choo-persist')
+    var logger = require('choo-log')
+    app.use(persist())
+    app.use(logger())
+  }
+  app.use(expose())
+  app.use(todoStore())
+
+  app.route('/', mainView)
+  app.route('#active', mainView)
+  app.route('#completed', mainView)
+  app.mount('body')
```

Comment:
oh, interesting. I didn't realise it was an either/or. I assumed you had to `mount` _and_ `start`. k thx